docs/managed_package.rst

```diff
@@ -1,27 +1,155 @@
-Release a managed package
+Release a Managed Package
 =========================
 
+This section outlines how to release a first generation (1GP) Salesforce Managed Package project. Salesforce.org's Release Engineering team practices `CumulusCI Flow<TODO link>`_, which incorporates all of these steps.
+
+
+
 Prerequisites
 -------------
 
-Deploy to a packaging org
+This section assumes:
+
+* CumulusCI is installed on your computer.
+* A Salesforce Managed Package project has been configured for use with CumulusCI.
+* A packaging org is connected to CumulusCI under the name of ``packaging``.
+
+
+
+Deploy to a Packaging Org
 -------------------------
 
-Upload and test a beta version
-------------------------------
+CumulusCI enables you to deploy to your ``packaging`` org.
+
+.. warning::
+
+    The ``ci_master`` flow runs the `uninstall_packaged_incremental<TODO>`_ task, which deploys destructive changes to the target org.
+
+.. code-block:: console
+
+    $ cci flow run ci_master --org packaging
+
+.. note::
+
+    This flow is `source format agnostic<TODO>`_.
+
+The ``ci_master`` flow executes these tasks in the target org.
+
+* Updates any project dependencies
+* Deploys any unpackaged metadata located in the ``pre`` directory
+* Deploys package metadata
+* Deploys destructive changes to remove metadata in the target org that is no longer in the local workspace
+
+
+
+Create a Beta Version
+---------------------
+
+The ``release_beta`` flow groups the common tasks that must be executed for the release of a new beta version of a project.
+
+.. code-block:: console
+
+    $ cci flow run release_beta --org packaging
+
+This flow typically runs against the project's ``packaging`` org where it:
+
+* Uploads a new beta version to the ``packaging`` org
+* Creates a new GitHub release tag for the new beta version
+* `Generates Release Notes<TODO#anchor>`_
```

Comment:
Can't make this a natural anchor link (for the Generate Release Notes subsection) without fudging the tense on every bullet point in this section. Is there another way to manipulate the anchor link here?

Reply:
I think we might have to use [cross references](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations). I'm going to look into this more later this week or early next once we get (most of) the content out of the way.